rpm.pc.in

```diff
@@ -8,7 +8,7 @@ Name: RPM
 Description: RPM Package Manager
 Version: @VERSION@
 URL: http://rpm.org
-# Requires: 
+Requires.private: @ZSTD_REQUIRES@
```

Comment:
well, we do for Libraries.private... but since we use pkg-config for zstd, it is better to put into Requires.private..